.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: go
+
+go:
+  - "1.12.5"
+
+go_import_path: github.com/kubeflow/kfserving
+
+install:
```

Comment:
What's executing this travis script? Where is this running? Seems very strange and hacky, whatever it's doing.

Reply:
github repo .travis.yaml can trigger triais build.
just follow how kubeflow/katib run "make test" as [here](https://github.com/kubeflow/katib/blob/master/.travis.yml)

Reply:
Are we moving to travis ? I guess I am not clear with the intention of this PR.

Reply:
in fact we doesn't need travis since current test flow doesn't take too long. So remove travis now.
this PR just fix test error